tests/scripts/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```

Comment:
Can this just be a symlink? I don't see any differences from the original script.

Reply:
Moved to tests/scripts as wrapper script has been added